Gems/Atom/RPI/Code/Source/RPI.Public/RPIUtils.cpp

```diff
@@ -110,6 +110,26 @@ namespace AZ
                 return ((value - origMin) / (origMax - origMin)) * (scaledMax - scaledMin) + scaledMin;
             }
 
+            float ScaleSNorm8Value(float value)
+            {
+                // Scale the value from AZ::s8 min/max to -1 to 1
+                // We need to treat -128 and -127 the same, so that we get a symmetric
+                // range of -127 to 127 with complementary scaled values of -1 to 1
+                constexpr float signedMax = static_cast<float>(std::numeric_limits<AZ::s8>::max());
+                constexpr float signedMin = -signedMax;
+                return ScaleValue(AZStd::max(value, signedMin), signedMin, signedMax, -1.0f, 1.0f);
```

Comment:
I wonder why you only make sure the value not smaller than signedMin but not 'not larger than signedMax' since the input value type is float.

Maybe it's better to use int8_t as input rather than float.

Reply:
Fair point, I didn't change the existing logic here, but I _did_ put it in a function that takes in a float. I'll change the input types to int8_t and int16_t.